model_analyzer/model_manager.py

```diff
@@ -157,6 +168,9 @@ def _run_model_config_sweep(self,
 
         next_model = model
         while True:
+            if self._state_manager.exiting():
```

Comment:
Will this `while true` conflict with the top level while true loop in the analyze.pl file

Reply:
Conflict? How do you mean?

Reply:
Are you sure you will get an "exiting()" statement? Any `while true` should be a code smell and may result in an infinite while loop. Can this be better served as `while !self._state_manager.exiting():` for example.

Reply:
There is no `do while` construct in python so usually the `while True: if ... break` is what is used. Here you may not get an `exiting()` but the  `get_model_sweep` will eventually return an empty list, which indicates the end of the sweep.